frontend/src/components/admin/AssignmentPanel.tsx

```
@@ -135,15 +130,12 @@ export function AssignmentPanel() {
       <Typography color="text.secondary" sx={{ mb: 3 }}>
         Assign experiences to regions based on spatial containment. Experiences are assigned to
         regions that contain their location point, and the assignment is propagated to ancestor regions.
+        A sync already places whatever moved during the run, so this is for the case a sync cannot
+        cover: region boundaries themselves changed, and every location has to be tested against them
+        again. It clears this world view's automatic assignments before rebuilding, so regions look
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)